app/javascript/controllers/maps_controller.js

```diff
@@ -0,0 +1,85 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = ["map", "field", "getCurrentLocation"]
+  static values = { latitud: Number,
+                    longitud: Number}
```

Comment:
I suggest these improvements for the readability, let me know if you agree or disagree?
```suggestion
  static values = {
    latitude: Number,
    longitude: Number,
  }
```

Reply:
Agree with this change!